cypress/e2e/dashboard/events/[eventId].cy.ts

```diff
@@ -38,12 +38,12 @@ describe('event dashboard', () => {
 
       cy.get<string>('@userName').then((userName) => {
         cy.get('@waitlist').not(`:contains(${userName})`);
-        cy.get('[data-cy=rsvps]').contains(userName);
+        cy.get('[data-cy=attendees]').contains(userName);
       });
 
       cy.get('@email')
         .mhGetSubject()
-        .should('include', 'Your RSVP is confirmed');
+        .should('include', 'Your attendance is confirmed');
```

Comment:
Actually, I thought people would use an extension like Multiple cursor case preserve, or something to make swapping them automatic.

But you have actually focused on details 👏 👍